events/open-source-marathon/index.html

```diff
@@ -6,6 +6,8 @@
     <title>Open Source Marathon</title>
     <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700" rel="stylesheet" />
     <link href="https://fonts.googleapis.com/css?family=Orbitron:400,500,700" rel="stylesheet" />
+    <link rel="stylesheet" href="../../assets/css/WshopImg.css">
+    <link rel="stylesheet" href="../../assets/css/stats.css">
```

Comment:
Make it absolute path
```suggestion
    <link rel="stylesheet" href="/assets/css/stats.css">
```